presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -296,6 +296,7 @@ public void testAggregationPushdown()
         // TODO support aggregation over expressions
 
         assertPushedDown("SELECT count(*) FROM nation");
+        assertPushedDown("SELECT count(1) FROM nation");
```

Comment:
Can you add one with an empty count too, `count()`?

Reply:
Adding empty count makes test failed. Presto supports empty count but PostgreSQL (or JDBC?) seems not.

```
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT COUNT()[*] FROM NATION"; expected "*, DISTINCT, ALL, NOT, EXISTS, INTERSECTS, UNIQUE"; SQL statement:
SELECT count() FROM nation [42001-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.getSyntaxError(DbException.java:243)
	at org.h2.command.Parser.getSyntaxError(Parser.java:1053)
	at org.h2.command.Parser.readTerm(Parser.java:4479)
```

Reply:
The error comes from H2, which is used as a reference
For now you can add a second param like this, to use a different query for H2:
```
assertPushedDown("SELECT count() FROM nation",  "SELECT count(*) FROM nation");
```

This won't be needed after https://github.com/prestosql/presto/pull/4466

Reply:
No, is it the limitation of H2 JDBC driver?

Reply:
@findepi Thanks a lot! It did work as we expected. I didn't know such usage of `assertPushedDown`.

Reply:
#4466 was merged so you may need a rebase